### DISCUSSION TITLE
View System Power Draw

### DISCUSSION THREAD

**Discussion by James-Aric** [FakeFishGames/Barotrauma#6584] [Ideas and suggestions]
It would be very nice to be able to view the power draw of a specific system when viewing the wiring, as it would allow for engineers to figure out what systems are pulling the most power, as well as give information towards what Sub upgrades for power efficiency should be bought. This information as far as I know, can only really be viewed in the Sub Editor, or by eyeballing the load in the reactor UI and making estimates based on what systems are active.

**Comment by James-E-A** [comment]
This could possibly be resolved in a DIY fashion by [`load_out` and `power_out` pins on Junction Boxes or Relays](https://github.com/Regalis11/Barotrauma/discussions/6794)

**Reply by James-Aric** [reply]
True, but with a lot of subs, there is limited wall space at best. It would be nice to open up the breaker for a machine and check the draw directly

**Reply by StruggleDuck** [reply]
I believe that the primary implementation for this would be while in the sub editor, and not in the middle of a campaign

**Comment by ignomen** [comment]
A relevant PR is here: https://github.com/Regalis11/Barotrauma/pull/7303
